### PR TITLE
fix qiita #2791

### PR DESCRIPTION
**Pull request by antgonza**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,46 +1,48 @@
...
+  # starting nginx
+  - mkdir /home/travis/miniconda3/envs/qiita/var/run/nginx/
+  - wget https://github.com/biocore/qiita/archive/dev.zip -O qiita-dev.zip
+  - unzip qiita-dev.zip
+  - nginx -c ${PWD}/qiita-dev/qiita_pet/nginx_example.conf
   - qiita-env make --no-load-ontologies
   - source deactivate
   # installing this plugin
-  - conda create --quiet --yes -n deblur-1.1.0 python=$PYTHON_VERSION pip nose flake8 h5py
+  - conda create --quiet --yes -n deblur-1.1.0 python=3.5 pip
```
Comment (charles-cowart) [inline .travis.yml]: python=3.6?

File: qp_deblur/deblur.py [hunk 2]
```diff
@@ -368,6 +369,20 @@ def deblur(qclient, job_id, parameters, out_dir):
     artifact_info = qclient.get("/qiita_db/artifacts/%s/" % artifact_id)
     fps = artifact_info['files']
 
+    # Getting preparation information
+    prep_info = qclient.get(
+        '/qiita_db/prep_template/%s/' % artifact_info['prep_information'][0])
+    df = pd.read_csv(prep_info['prep-file'], sep='\t')
+    if 'platform' not in [x.lower() for x in df.columns]:
+        error_msg = ('Preparation Information File does not have a platform '
+                     'column, which is requiered')
```
Comment (charles-cowart) [inline qp_deblur/deblur.py]: requiered -> required

File: .travis.yml (hunk 1 above)
Comment (antgonza) [reply .travis.yml]: there is a hard dependency on python 3.5 for fragment insertion so can't, [see](https://travis-ci.org/qiita-spots/qp-deblur/builds/519398036?utm_source=github_status&utm_medium=notification).

File: qp_deblur/deblur.py (hunk 2 above)
Comment (antgonza) [reply qp_deblur/deblur.py]: thanks

**Pull request merged**